app/bundles/CoreBundle/Resources/views/Notification/notification.html.twig

```diff
@@ -7,7 +7,13 @@
             {% endif %}
         </div>
         {% if n.header %}
-            <div class="notification-header fs-b-e fw-b mb-3 mt-sm">{{ format(n.header)|trans }}</div>
+            <div class="notification-header fs-b-e fw-b mb-3 mt-sm">
+                {% if 'mautic.' in n.header %}
+                    {{ n.header | trans }}
+                {% else %}
+                    {{ format(n.header, 'html') }}
+                {% endif %}
+            </div>
```

Comment:
I'd suggest
```suggestion
           <div class="notification-header fs-b-e fw-b mb-3 mt-sm">{{ n.header|trans|purify }}</div>
```
What do you think? The Mautic's `format` twig function is not as advanced in catching harmful HTML as the Purify library.

Reply:
@Hugo-Prossaird ?